server/service/osquery_utils/queries.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/spf13/cast"
 )
 
+// Some machines don't have a correctly set serial number, create a default ignore list
+// https://github.com/fleetdm/fleet/issues/25993
+var invalidHardwareSerialRegexp = regexp.MustCompile("(?i)(?:default|serial|string)")
```

Comment:
Probably not necessary to put this in a code comment, but for the record can you say how you formulated this ignore list?  I see in the attached ticket that some systems were returning "default string" but I don't see "serial" referenced.

Reply:
The first image in the thread has a serial number called `System Serial Number`

Reply:
I left the comment as a reference to where I got the words from, since there seems to be absolutely no standard for these things